front/README.md

```diff
@@ -45,21 +45,28 @@ It requires:
 
 Now you can run the test with `cd front/ && yarn e2e-tests`.
 
-> [!CAUTION]
-> If you try to run `yarn start` instead of running it through docker, you'll notice it doesn't
-> work because the gateway can't access your local port from inside a container. 2 solutions:
+If you are using a linux distribution not supported by playwright (which only supports windows, macos and ubuntu/debian),
```

Comment:
```suggestion
If you are using a Linux distribution not supported by playwright (which only supports Windows, MacOS and Ubuntu/Debian),
```

Reply:
Actually the correct capitalization is "macOS" :P

Reply:
Fix'd :p

Reply:
Done, thanks